src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
```

Comment:
- [ ] For now, I am trying to add a book but cannot see it on the UI please  try to fix that and display the books on the `UI` also remove to add the possibility to remove a book

https://user-images.githubusercontent.com/75126481/155113289-7e503fe0-4ac6-4114-a7af-9e37965a06dd.MP4